docs/learn/learn-parachains-protocol.md

```diff
@@ -448,10 +448,10 @@ there are approval checks (described above) and disputes. The latter ensures tha
 include something invalid is caught and the offending validators are punished.
 
 Signed negative statements will lead to a dispute, and if there are false negatives, whoever will be
```

Comment:
Let's focus on the classic disputes and not cover the false negatives here.

Reply:
remvoved